### PR TITLE
feat(ci): Run frontend test and lint with gh actions

### PR DESCRIPTION
**Pull request by fliiiix**
Maybe we should test and lint js code in ci

### CODE REVIEW

File: .github/workflows/docat.yml [hunk 1]
```diff
@@ -39,9 +39,33 @@ jobs:
...
           python -m poetry run pytest
 
+  javascript-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          #cache: 'yarn'
```
Comment (randombenj) [inline .github/workflows/docat.yml:49]: Why is this commented out?

I would suggest to either remove it or use it?

File: .github/workflows/docat.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: docat ci
 on: [push]
```
Comment (timofurrer) [inline .github/workflows/docat.yml:3]:
```suggestion
on: [push, pull_request]
```

I think we require a `pull_request` trigger, too.

**Review by timofurrer — APPROVED** [review]
Small thing unrelated to the PR. Otherwise, LGTM 🎉

File: .github/workflows/docat.yml (hunk 1 above)
Comment (fliiiix) [reply .github/workflows/docat.yml:49]: on purpose as a reminder that we could look into caching - but i can also remove it

File: .github/workflows/docat.yml (hunk 2 above)
Comment (fliiiix) [reply .github/workflows/docat.yml:3]: sure i checked the docs and idk lets add it

**Pull request merged**